aten/src/ATen/native/native_functions.yaml

```diff
@@ -4292,6 +4292,21 @@
   device_check: NoCheck
   device_guard: False
 
+- func: nested_tensor_size(Tensor self, int dim) -> int
```

Comment:
Trying to figure out why I need all of these. They are definitely nice to have and I have used for debugging from python... but the names are dumb and confusing. I also think `nested_tensor_size` is a reimpl of get_consistent_last_dim() so this should go. I still kind of like the last two to be exposed but maybe add a dispatch for nested_size(). could also only have nested_size_tensor and the user has to unbind.... decisions, decisions

Reply:
nested _size_tensor was needed in inputmeta data. But I think I can safely remove nested_tensor_size()